pkg/app/cassandra.go

```diff
@@ -47,17 +47,17 @@ type CassandraInstance struct {
 }
 
 // NewCassandraInstance returns new cassandra application
-// Last tested working version "0.13.3"
 func NewCassandraInstance(name string) App {
 	return &CassandraInstance{
 		name: name,
 		chart: helm.ChartInfo{
 			Release:  appendRandString(name),
-			RepoURL:  helm.BitnamiRepoURL,
-			RepoName: helm.BitnamiRepoName,
+			RepoURL:  helm.IncubatorRepoURL,
 			Chart:    "cassandra",
+			RepoName: helm.IncubatorRepoName,
+			Version:  "0.13.3",
 			Values: map[string]string{
-				"image.repository":    "kanisterio/cassandra",
+				"image.repo":    "kanisterio/cassandra",
```

Comment:
Does this need to be `image.repository`?

Reply:
And further checking of #755 shows that image.repo is the correct revert. Removing hold-off label.

Reply:
But... this file fail travis because it needs a gofmt.

Reply:
I will ask @arush-sal to take a look into this.